UPDATING.md

```diff
@@ -57,6 +57,26 @@ https://developers.google.com/style/inclusive-documentation
 
 -->
 
+### Cli use exactly single character for short option
+
+Use exactly one single character for short option in Airflow cli, New commands are available according to the following table:
+
+| Old command                                        | New command                                       |
+|----------------------------------------------------|---------------------------------------------------|
+| ``airflow (dags|tasks|scheduler) [-sd, --subdir]`` | ``airflow (dags|tasks|scheduler) [-S, --subdir]`` |
+| ``airflow tasks test [-dr, --dry_run]``            | ``airflow tasks test [-dr, --dry_run]``           |
+| ``airflow dags backfill [-dr, --dry_run]``         | ``airflow dags backfill [-d, --dry_run]``         |
+| ``airflow tasks clear [-dx, --dag_regex]``         | ``airflow tasks clear [-R, --dag_regex]``         |
+| ``airflow kerberos [-kt, --keytab]``               | ``airflow kerberos [-k, --keytab]``               |
+| ``airflow tasks run [-int, --interactive]``        | ``airflow tasks run [-N, --interactive]``         |
+| ``airflow webserver [-hn, --hostname]``            | ``airflow webserver [-H, --hostname]``            |
+| ``airflow celery worker [-cn, --celery_hostname]`` | ``airflow celery worker [-H, --celery_hostname]`` |
+| ``airflow celery flower [-hn, --hostname]``        | ``airflow celery flower [-H, --hostname]``        |
+| ``airflow celery flower [-fc, --flower_conf]``     | ``airflow celery flower [-c, --flower_conf]``     |
```

Comment:
-c is already used.
```
        'conf': Arg(
            ('-c', '--conf'),
            "JSON string that gets pickled into the DagRun's conf attribute"),

```

Reply:
But those for diff subcommand, do you think we should use unique short option in our cli? I find out if arg conflict our document build pre-test in Travis-CI will failed.

Reply:
Same command is fine to be used in different sub-commands

Reply:
Yeah, I think so, what do you think @mik-laj